id-verifier/Cargo.lock

```diff
@@ -604,7 +604,7 @@ checksum = "acbf1af155f9b9ef647e42cdc158db4b64a1b61f743629225fde6f3e0be2a7c7"
 
 [[package]]
```

Comment:
bumping the version of the `id-verifier`?

Reply:
Did not have a changelog either, I bumped it now and added the changelog.